docs/README.md

```diff
@@ -22,37 +22,37 @@ plugins:
     type: elastic_output
 ```
 
-## What is a plugin?
-A plugin is the most basic unit of log monitoring. Each plugin fulfills only a single responsibility, such as reading lines from a file, or parsing JSON from a field. These plugins are then chained together in a pipeline to achieve a desired result.
+## What is an operator?
+An operator is the most basic unit of log tranformation. Each operator fulfills only a single responsibility, such as reading lines from a file, or parsing JSON from a field. These plugins are then chained together in a pipeline to achieve a desired result.
```

Comment:
nit: Not all operators transform a log. Maybe `processing` instead of `transformation`?